src/storage/rdb.cc

```diff
@@ -125,7 +125,7 @@ StatusOr<uint64_t> RDB::loadObjectLen(bool *is_encoded) {
         GET_OR_RET(stream_->Read(reinterpret_cast<char *>(&len), sizeof(uint64_t)));
         return ntohu64(len);
       } else {
-        return {Status::NotOK, fmt::format("Unknown RDB string encoding type {} byte {}", type, c)};
+        return {Status::NotOK, fmt::format("Unknown RDB co -string encoding type {} byte {}", type, c)};
```

Comment:
Is there any special meaning of `co -`?

Reply:
No, I wrongly typed the keyboard and submit the change.

Reply:
Fixed now.